tests/core/test_notifications.py

```diff
@@ -22,161 +22,201 @@
+        user=qualified_volunteer,
+    ).form
+    form["individual_start_time_1"] = "07:42"
+    form.submit(name="signup_choice", value="customize").follow()
+
+    # assert only notification of the correct type exist
+    assert set(Notification.objects.all().values_list("slug", flat=True)) == {
+        ResponsibleConfirmedParticipationCustomizedNotification.slug
+    }
+    plaintext = ResponsibleConfirmedParticipationCustomizedNotification.get_body(
+        Notification.objects.first()
+    )
+    assert "7:42" in plaintext
+    call_command("send_notifications")
+    assert not Notification.objects.filter(processing_completed=False).exists()
+
+
+def test_participant_participation_customized_notification(
+    django_app, event, planner, qualified_volunteer
+):
+    participation = LocalParticipation.objects.create(
+        shift=event.shifts.first(),
+        user=qualified_volunteer,
+        state=AbstractParticipation.States.CONFIRMED,
+    )
+    # change individual start time
+    response = django_app.get(
+        reverse("core:shift_disposition", kwargs=dict(pk=participation.shift.pk)),
+        user=planner,
+    )
+    form = response.forms["participations-form"]
+    form["participations-0-individual_start_time_1"] = "07:42"
+    form.submit()
+
+    # assert only notification of the correct type exist
+    assert Notification.objects.get().slug == ParticipationCustomizationNotification.slug
+
+    plaintext = ParticipationCustomizationNotification.get_body(Notification.objects.first())
+    assert "7:42" in plaintext
+    call_command("send_notifications")
+    assert not Notification.objects.filter(processing_completed=False).exists()
+
+
+def test_responsibles_dont_get_notified_about_own_disposition_action(
+    django_app, event, planner, qualified_volunteer
+):
+    participation = LocalParticipation.objects.create(
+        shift=event.shifts.first(),
+        user=qualified_volunteer,
+        state=AbstractParticipation.States.REQUESTED,
+    )
+    # change individual start time
+    response = django_app.get(
+        reverse("core:shift_disposition", kwargs=dict(pk=participation.shift.pk)),
+        user=planner,
+    )
+    form = response.forms["participations-form"]
+    form["participations-0-state"] = AbstractParticipation.States.CONFIRMED
+    form.submit()
+
+    # the planner that did the disposition should not be notified about their own action
+    call_command("send_notifications")
+    for m in mail.outbox:
+        assert "Luisa Durchblick <luisa@localhost>" not in m.to
+
+
+def test_responsibles_dont_get_notified_about_own_signup(django_app, event, planner):
+    django_app.get(
+        event.get_absolute_url(),
+        user=planner,
+    ).form.submit(name="signup_choice", value="sign_up").follow()
+    # even if responsible, the planner that signed themselves up should not be notified about their own action
+    call_command("send_notifications")
+    for m in mail.outbox:
```

Comment:
These are the two new tests. The rest was moved out of the class that didn't do anything.